cmd/fetch_repo/module.go

```diff
@@ -93,9 +94,11 @@ func fetchModule(dest, importpath, version, sum string) error {
 	// Parse the JSON output.
 	var dl struct{ Dir, Sum, Error string }
 	if err := json.Unmarshal(buf.Bytes(), &dl); err != nil {
+		_, _ = os.Stderr.Write(bufErr.Bytes())
 		return err
 	}
 	if dl.Error != "" {
+		_, _ = os.Stderr.Write(bufErr.Bytes())
```

Comment:
If `go mod download` writes out JSON, it won't write anything useful on stderr (maybe download chatter). So let's drop this line.